crates/vm/levm/src/opcode_handlers/system.rs

```diff
@@ -364,25 +292,26 @@ impl VM {
         current_call_frame: &mut CallFrame,
     ) -> Result<OpcodeSuccess, VMError> {
         let value_in_wei_to_send = current_call_frame.stack.pop()?;
-        let code_offset_in_memory: usize = current_call_frame
+        let code_offset_in_memory = current_call_frame.stack.pop()?;
+        let code_size_in_memory: usize = current_call_frame
             .stack
             .pop()?
             .try_into()
             .map_err(|_err| VMError::VeryLargeNumber)?;
-        let code_size_in_memory = current_call_frame
-            .stack
-            .pop()?
+
+        let new_size: usize = code_offset_in_memory
```

Comment:
In `op_create` we also need to handle the scenario in which the offset is greater than an `usize` and the size is 0.
For calculating new memory size we have to make use of `calculate_memory_size()` like in the other opcodes, right?

Reply:
Done in [7d306fa](https://github.com/lambdaclass/ethrex/pull/1458/commits/7d306fae02205b1461716c08e2c12338a7176741)